lib/systems/include/Controllers/LoadCellVectoringController.h

```diff
@@ -57,12 +57,20 @@ class TorqueControllerLoadCellVectoring : public virtual Controller
         writeout_.command = BaseControllerParams::TC_COMMAND_NO_TORQUE;
         writeout_.ready = false;
     }
+    /// @brief default contructor with balanced dfault values: rearTorqueScale = 1.0, regenTorqueScale = 1.0
     TorqueControllerLoadCellVectoring() : TorqueControllerLoadCellVectoring(1.0, 1.0) {}
 
+    /// @brief Calculates speed set point based on normal force applied to wheels individually
+    /// @param SysTick_s &tick 
+    /// @param PedalsSystemData_s &pedalsData 
+    /// @param LoadCellInterfaceOutput_s &loadCellData 
     void tick(
         const SysTick_s &tick,
         const PedalsSystemData_s &pedalsData,
         const LoadCellInterfaceOutput_s &loadCellData);
+    /// @brief ticks controller
```

Comment:
Why do we need to re-define this function here if it should just be implementing a function from the parent class?
If documentation is identical as an inherited function, don't copy-paste it-- instead, just put a note to refer to parent class BaseController.